src/lib/homeobject_impl.hpp

```diff
@@ -32,11 +34,19 @@ struct Shard {
 using ShardList = std::list< Shard >;
 using ShardIterator = ShardList::iterator;
 
+struct PGReplDevBase {};
```

Comment:
?

Reply:
Tried something else, but forgot to remove. Thanks!